templater.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+class Templater {
+    constructor() {
+        this.tagsArray = {};
```

Comment:
We should not use type of variable in its name, so it's enough to name this var as `tags`